src/app/pages/administration/jobs/jobs.jsx

```diff
@@ -28,17 +29,23 @@ import theme from 'themes/theme';
 
 const useStyles = makeStyles({
   container: {
-    display: 'grid',
+    display: 'flex',
     gap: '16px',
+    flexDirection: 'column',
     gridTemplateColumns: '1fr',
-    [theme.breakpoints.up('sm')]: {
+    [theme.breakpoints.up('lg')]: {
       gridTemplateColumns: '1fr 1fr',
+      display: 'grid',
```

Comment:
I can't tell what's happening here. Is this a flex display at small scales, and a grid at large ones? If so, why is there a gridTemplateColumns at small scales? That said, do we have to do it this way? I feel like `display: flex` is the right choice for small elements inside larger ones, but `display: grid` is our tool for laying out an entire page / component, and this `container` element should be the full page/component, right?

Reply:
We need to talk about this so we're on the same path